src/main/java/com/jvm_bloggers/frontend/common_components/infinite_scroll/InfiniteScrollingBehavior.java

```diff
@@ -0,0 +1,122 @@
+package com.jvm_bloggers.frontend.common_components.infinite_scroll;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import javaslang.control.Try;
+import lombok.extern.slf4j.Slf4j;
+
+import org.apache.wicket.Component;
+import org.apache.wicket.behavior.Behavior;
+import org.apache.wicket.markup.head.IHeaderResponse;
+import org.apache.wicket.markup.head.JavaScriptHeaderItem;
+import org.apache.wicket.markup.head.OnLoadHeaderItem;
+import org.apache.wicket.request.resource.ResourceReference;
+import org.apache.wicket.resource.JQueryPluginResourceReference;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Slf4j
+public class InfiniteScrollingBehavior extends Behavior {
+
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+    private static final ResourceReference
+        JS = new JQueryPluginResourceReference(InfiniteScrollingBehavior.class,
+        "js/jquery.infinitescroll.js");
```

Comment:
That would be the best way to go but unfortunately, I had to modify JS to adjust it to Wicket URLs (pagination)